### PR TITLE
[fix bug 1344270] Open Innovation sub page for Internet Health

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Adds Internet Health sub page for Open Innovation
- Strings are not yet confirmed as final
- Theme color and graphics are still TBD.

Note: this PR uses the existing `internet_health_digital_inclusion` l10n tag, as these string changes are to be batched up and sent out alongside the Digital Inclusion sub page.

## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1344270

## Checklist
- [x] Requires l10n changes.
- [ ] Related functional & integration tests passing.

### CODE REVIEW

**Comment by alexgibson** [comment]
Ok I believe everything here should now be final.

**Comment by alexgibson** [comment]
There's one more icon to be updated here, but no other changes

**Comment by alexgibson** [comment]
Final icon is now in place.

**Comment by alexgibson** [comment]
CC @peiying2 @flodolo - strings are now final for this, good for l10n (reminder that this is using the same l10n tag as we used for the inclusion page).

**Comment by alexgibson** [comment]
I just realised, we still need the meta page image - are you able to put this together with the template you made @craigcook?

**Comment by craigcook** [comment]
> I just realised, we still need the meta page image - are you able to put this together with the template you made @craigcook?

I've made an image and attached it to the bug.

File: bedrock/mozorg/templates/mozorg/internet-health/open-innovation.html [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+              </p>
+            </li>
+            <li>
+              <h4>{{ _('Advocating for protections') }}</h4>
+              <p>
+              {% trans internet_users='https://blog.mozilla.org/netpolicy/2016/03/07/challenges-to-openness-under-u-s-copyright-law/',
+                       creators='https://blog.mozilla.org/netpolicy/2017/02/21/we-are-all-creators-now/',
+                       intermediaries='https://blog.mozilla.org/netpolicy/2016/04/05/reining-in-abuses-of-the-dmca-notice-system/'
+              %}
+                Advocating for clear protections for <a href="{{ internet_users }}">internet users</a>,
```
Comment (flodolo) [inline bedrock/mozorg/templates/mozorg/internet-health/open-innovation.html]: Copy is using Internet so far, I don't think the switch to "internet" is wanted.

**Comment by alexgibson** [comment]
@craigcook - meta image added, @flodolo copy updates made - thanks.

**Pull request merged**